ppr-ui/src/store/state/state-model.ts

```diff
@@ -39,6 +40,7 @@ export const stateModel: StateModelIF = {
     },
     registrationNumber: '',
     registrationType: null,
+    registrationFlowType: RegistrationFlowType.NEW,
```

Comment:
can you set it to this inside *mutateNewRegistration* as well? Just in case they do a discharge or a renewal and then do a new registration afterwards

Reply:
done